pkg/clients/serviceaccountpolicy/serviceaccountpolicy.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2020 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package serviceaccountpolicy
+
+import (
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"github.com/mitchellh/copystructure"
+	"github.com/pkg/errors"
+	"google.golang.org/api/iam/v1"
+
+	"github.com/crossplane/provider-gcp/apis/iam/v1alpha1"
+)
+
+const (
+	// https://cloud.google.com/kms/docs/reference/rest/v1/Policy
+	// Specifies the format of the policy.
+	// Any operation that affects conditional role bindings must specify version 3.
+	// Our CR supports conditional role bindings.
+	policyVersion = 3
```

Comment:
noticed this is copied a few places, does it make sense to move it to a common package?

Reply:
Good point, done 👍